wireless_modules/mqtt_client.py

```diff
@@ -70,3 +70,9 @@ def wait_for_message(self):
         This function blocks until a message is received on one of the subscribed topics
         """
         self.client.wait_msg()
+
+    def disconnect(self):
```

Comment:
We don't seem to actually use this disconnect function anywhere. Did you mean to use it for something?

Reply:
I realised MQTT allows you to disconnect from a broker as well. I don't see a use of it right now, but in case we need it in the future thought I'll put it in this class (especially since the main aim with this PR was just to get this class finalised).

Reply:
Yeah that makes sense, good thinking!